x-pack/plugins/apm/public/components/routing/templates/settings_template.tsx

```diff
@@ -102,6 +102,8 @@ function getTabs({
           kuery: '',
           agentLanguage: '',
           serviceName: '',
+          rangeFrom: 'now-24h',
```

Comment:
The ranges persist in the URL while you navigate.  Passing default in setting templates will affect the other pages after visiting agent explorer. As a result, the user might experience a performance issues, esp if they have big data set

Reply:
As far as I see, the other routes needing the time range are guarded with `RedirectWithDefaultDateRange` top-level component.
<img width="762" alt="image" src="https://user-images.githubusercontent.com/1313018/203778478-33e93c8d-93a1-444e-b3d5-b96c994f13fb.png">
<img width="768" alt="image" src="https://user-images.githubusercontent.com/1313018/203778098-e218b9bc-200b-4fc9-a86c-98ae2c402d63.png">

I also checked the behaviour here

https://user-images.githubusercontent.com/1313018/203778174-b68db466-ebf9-495d-ac74-c277c9ce6936.mov

Reply:
In any case I will move this commit to another PR

Reply:
Thank you, I was able to reproduce it